assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -79,12 +80,13 @@ class PageEdit extends Component {
 			this.props.setAttributes( { anchor: getUniqueId() } );
 		}
 
+		this.onSelectMedia = this.onSelectMedia.bind( this );
```

Comment:
Why was this moved?

Reply:
I was originally trying to get rid of the binding and use class properties instead (see #3152), and then it came out that class properties are not configured at this moment, so added it back to a different place, no difference otherwise.